src/main/resources/niceload.accessWidener

```diff
@@ -1,4 +1,6 @@
 accessWidener	v1	named
 accessible	field	net/minecraft/client/gui/screen/SplashOverlay	MOJANG_RED	I
 accessible	field	net/minecraft/client/gui/screen/SplashOverlay	MONOCHROME_BLACK	I
+mutable	    field	net/minecraft/client/gui/screen/SplashOverlay	MOJANG_RED	I
+mutable  	field	net/minecraft/client/gui/screen/SplashOverlay	MONOCHROME_BLACK	I
```

Comment:
`MONOCHROME_BLACK` is never mutated, so it doesn't need a `mutable` AW.

Reply:
Okay.